module/zfs/zcp_get.c

```diff
@@ -693,9 +693,9 @@ parse_written_prop(const char *dataset_name, const char *prop_name,
 	ASSERT(zfs_prop_written(prop_name));
 	const char *name = prop_name + ZFS_WRITTEN_PROP_PREFIX_LEN;
 	if (strchr(name, '@') == NULL) {
-		(void) sprintf(snap_name, "%s@%s", dataset_name, name);
+		(void) snprintf(snap_name, sizeof(name), "%s@%s", dataset_name, name);
```

Comment:
You want `ZFS_MAX_DATASET_NAME_LEN` here instead of `sizeof(name)`.

Reply:
Woah, that's just concerning, had to go check old port but it was correct there - phew.